Bloxstrap/Integrations/ActivityWatcher.cs

```diff
@@ -284,9 +284,11 @@ public async Task<string> GetServerLocation()
 
             try
             {
-                locationCity = await App.HttpClient.GetStringAsync($"https://ipinfo.io/{ActivityMachineAddress}/city");
-                locationRegion = await App.HttpClient.GetStringAsync($"https://ipinfo.io/{ActivityMachineAddress}/region");
-                locationCountry = await App.HttpClient.GetStringAsync($"https://ipinfo.io/{ActivityMachineAddress}/country");
+                locationInformation = await Http.GetJson<IPInfoResponse>($"https://ipinfo.io/{ActivityMachineAddress}/json");
```

Comment:
`locationInformation` is not properly initialised and will cause an error on build.